packages/mermaid/src/docs/syntax/flowchart.md

```diff
@@ -5,11 +5,9 @@ outline: 'deep' # shows all h3 headings in outline in Vitepress
 
 # Flowcharts - Basic Syntax
 
-All Flowcharts are composed of **nodes**, the geometric shapes and **edges**, the arrows or lines. The mermaid code defines the way that these **nodes** and **edges** are made and interact.
+Flowcharts are composed of **nodes** (geometric shapes) and **edges** (arrows or lines). The Mermaid code defines how nodes and edges are made and accommodates different arrow types, multi-directional arrows, and any linking to and from subgraphs.
 
-It can also accommodate different arrow types, multi directional arrows, and linking to and from subgraphs.
-
-> **Important note**: Do not type the word "end" as a Flowchart node. Capitalize all or any one the letters to keep the flowchart from breaking, i.e, "End" or "END". Or you can apply this [workaround](https://github.com/mermaid-js/mermaid/issues/1444#issuecomment-639528897).
+> **Important**: If you are using the word "end" in a Flowchart node, capitalize the entire word or any of the letters (e.g., "End" or "END"), or apply this [workaround](https://github.com/mermaid-js/mermaid/issues/1444#issuecomment-639528897). Typing "end" in all lowercase letters will break the Flowchart.
```

Comment:
```suggestion
```warning
If you are using the word "end" in a Flowchart node, capitalize the entire word or any of the letters (e.g., "End" or "END"), or apply this [workaround](https://github.com/mermaid-js/mermaid/issues/1444#issuecomment-639528897). Typing "end" in all lowercase letters will break the Flowchart.
\``` Remove the \ here. Suggestion didn't include the ending triplequote.
```

Reply:
@sidharthv96 

The yellow text on yellow background is difficult to read:

<img width="750" alt="Screenshot 2023-04-19 at 12 13 28 PM" src="https://user-images.githubusercontent.com/35910788/233177023-913d3229-986d-4f9f-97a5-be5dd4c9e337.png">

Reply:
I agree, We should fix that with style changes in a later PR as it'll affect the whole site. 

But, if we don't use the proper warning format here, we'll forget to add it once the style is fixed.

Reply:
<img width="733" alt="image" src="https://user-images.githubusercontent.com/10703445/233254923-0189ad57-2dd7-44a0-bf1c-048024b47476.png">

Vitepress warning looks fine. Maybe it's a different version.

Reply:
They have fixed it in the new version. I've upgraded vitepress in develop. 
<img width="745" alt="image" src="https://user-images.githubusercontent.com/10703445/233255369-a73a1678-b2e6-4649-9022-08c40929a354.png">